src/main/js/tansaction/transaction-preview.js

```diff
@@ -0,0 +1,118 @@
+import React, { useEffect, useState } from "react";
+import Container from "react-bootstrap/Container";
+import { Button, Table } from "react-bootstrap";
+import TransactionModal from "./transaction-modal";
+import TransactionConfModal from "./transaction-conf-modal";
+
+/**
+ * The purpose of this method is to generate the bank account preview table.
+ *
+ * @author Xiaobing Hou
+ * @date 06/03/2022
+ */
+const TransactionPreview = (props) => {
+  const [selectedTransaction, setSelectedTransaction] = useState();
```

Comment:
Only the `TransactionContainer` should have state. All the children should simple receive data from the parent. So let's move up the state.

Reply:
done